dcos_installer/config.py

```diff
@@ -45,6 +45,20 @@ def normalize_config_validation(messages):
     return validation
 
 
+def normalize_config_validation_exception(error):
+    """
+    Accepts ValidationError which is transofmed to dict and processed by
+    `normalize_config_validation` function.
+
+    :param exception: An exception raised during the config validation
+    :type exception: ValidationError
```

Comment:
Please use Python's type hints instead of docstrings for type annotation. We use docstrings in a lot of places because DC/OS predates Python's support for type hints, but we're trying to favor type hints going forward.

https://github.com/dcos/dcos/blob/master/docs/styleguides/python.md
https://www.python.org/dev/peps/pep-0484/
https://docs.python.org/3/library/typing.html